mysite/psa/tests.py

```diff
@@ -816,3 +818,68 @@ def test_optional_expiry_interval_param(self):
         _wrapped_view = preview_access(decorated_view, expiry_interval=interval)
         _wrapped_view(self.request)
         self.request.session.set_expiry.assert_called_once_with(interval)
+
+
+class ForgotPasswordTest(TestCase):
+    """
+    Functional test reset password
+    """
+
+    def setUp(self):
+        self.user = User(username='user')
+        self.user.set_password('user')
+        self.user.save()
+        self.token = TokenForgotPassword()
```

Comment:
I think it will be better to save few lines )))

    self.token = TokenForgotPassword(user=self.user, next_url='/')
    self.token.save()